### PR TITLE
update icons' styling

### PR DESCRIPTION
**Pull request by unpWn4bL3**

### CODE REVIEW

File: src/shared/components/Documentation.tsx [hunk 1]
```diff
@@ -774,7 +774,11 @@ export const Pre = (props: PreProps) => {
   </pre>
 }
 
-const MobileAsideToggleButton = styled.div`
+interface isPhoneAsideVisible {
```
Comment (victorteokw) [inline src/shared/components/Documentation.tsx:777]: Let's rename this to `MobileAsideToggleButtonProps`

File: src/shared/components/DocumentationSidebar.tsx [hunk 2]
```diff
@@ -215,7 +215,11 @@ const SidebarWithToc: (props: { item: TocItem, path: string, phoneOpen?: boolean
   </DocSidebarContainer>
 }
 
-const SidebarToggleButton = styled.div`
+interface isSidebarVisible {
```
Comment (victorteokw) [inline src/shared/components/DocumentationSidebar.tsx:218]: Rename to `SidebarToggleButtonProps`

File: src/shared/components/Heading/index.tsx [hunk 3]
```diff
@@ -13,7 +13,11 @@ import SlackIcon from './SlackIcon'
 import XIcon from './XIcon'
 import PatreonIcon from './PatreonIcon'
 
-const HeadingMenuButton = styled.button`
+interface isShowMenu {
```
Comment (victorteokw) [inline src/shared/components/Heading/index.tsx:16]: `HeadingMenuButtonProps`

File: src/shared/components/Heading/index.tsx [hunk 4]
```diff
@@ -127,6 +151,8 @@ const HeadingNavItems = styled.ul`
   ${phone} {
     ${flexColumn('center')}
     align-items: flex-start;
+    position: absolute;
+    top: 8%;
```
Comment (victorteokw) [inline src/shared/components/Heading/index.tsx:155]: What does this 8% mean?

Comment (unpWn4bL3) [reply src/shared/components/Heading/index.tsx:155]: With out "top: 8%", page would look like this:
image

Comment (victorteokw) [reply src/shared/components/Heading/index.tsx:155]: Please given some pixels like `92px`

Comment (unpWn4bL3) [reply src/shared/components/Heading/index.tsx]: changed to 92px

**Review by victorteokw — APPROVED** [review]

**Pull request merged**